benchmark/test

```diff
@@ -0,0 +1,31 @@
+
+file_name:1:configs/dmc_sim_8_6.conf
```

Comment:
It's a sample config file that is required to run the benchmark script successfully. However, there are 3 output files named "test", "test.dat", and "test.pdf" which I removed from the commit.